### ISSUE TITLE
2016, July 25-26 - IPFS Lisbon Workshop: Q3 and Beyond

### ISSUE THREAD

**Issue by RichardLitt** [ipfs/community#142]
**tl:dr;** An IPFS Workshop and hackday, in Lisbon, July 25th-26th, accessible remotely, all invited, proposals for talks and events encouraged, travel scholarships available as needed.

---

A lot of exciting things have been happening in IPFS Land. Our talks at the [Decentralized Web summit](http://www.decentralizedweb.net/) give a good insight into that, what with [js-ipfs](https://github.com/ipfs/js-ipfs) being released,  and IPFS now working on the browser; [Orbit](https://github.com/haadcode/orbit) as a great example of an app running on js-ipfs; the idea of IPFS Internet Archives and using IPFS for Permanent Data Storage becoming more well known; and [libp2p](https://github.com/libp2p) as a project itself gaining traction.

Let’s tap into all of the momentum that has been building behind IPFS over the past couple months and get together to chart the course for IPFS into the third quarter this year, and beyond. There’s lots to demo, lots to talk about and discuss, lots to hack on, and most importantly lots to celebrate! :tada:

Anyone regularly attending or keeping track of our weekly sprints knows that the common theme has been closing out Q2. As things are drawing to a close on this chapter of the year, let’s take this energy and work together at a two day IPFS Workshop in Lisbon!
### IPFS Workshop: Q3 and Beyond July 25 - 26

**First off: Apologies for the short notice. We decide to do this a few days ago. We noticed a lot of the community [interested in hanging out in Lisbon](https://github.com/ipfs/community/issues/135), so we decided to do something (bigger than a meet up) together!**

**Define 'workshop':** Just as collaborative but less formal than a conference, just as intense but more structured than a hackday. Things you should anticipate taking place: discussions, demos, roadvmapping, hacking, bike shedding, and general hanging out as a community.

**What it is:** We’ve been identifying the big things that we want to tackle for the upcoming quarter, and will be having some per-project sessions beforehand (more info to come!). These sessions will be open for people interested in voicing needs or ideas, so that we can go into this workshop with a laundry list of dependencies, blockers, and features for IPFS that need to be addressed. We will spend time during the beginning of the workshop determining the best strategy to meet those needs, address and hack through any known blockers, and put together the road mapping strategy for IPFS, go-ipfs, js-ipfs, and other related sub-projects.

**Why we’re doing it:** As momentum behind IPFS is gathering, there’s a lot of energy, interest, and feedback to take into consideration when looking at the roadmap for the future, as well as many sub-projects to complete. We want to focus that momentum and the increased demand to make sure that we are targeting the best milestones.

**Who's going?** Everyone in the IPFS Community is invited! So far, we've confirmed several of the project leads. You can track who's coming in [this issue](https://github.com/ipfs/2016-Q3-Workshop/issues/1). It's going to be a blast, and it would be great to have as many of us present -- in person or calling in -- as we can!

**Travel Arrangements:** We are going to self-organize some AirBnBs to stay at. Post in [this issue](https://github.com/ipfs/2016-Q3-Workshop/issues/2) if you'd like to find some space together. Also, we know many people would love to come but are unable to due to sudden travel being unaffordable. If this is you, we have good news! We have some sponsorship funds lined up to offer as a "Need based scholarship". We have limited funds here so we will try to allocate it to regular community members who need it the most. Just answer a  [few short questions](https://docs.google.com/a/ipn.io/forms/d/1fccUfZPBhxagiw8OU0yR-1qNVQuVh9vI6_FioZcS_DI/edit?ts=577e7a78) and we’ll get back to you.

**Can't make it to Portugal physically? Make it virtually!** Physical attendance is entirely optional. This event is available to interact through Video calls (Zoom.us), hangouts, live IRC chats, and more EtherPads than all the sprints combined! \o/ Exact times TBD (still figuring out the timezone mess 😄 )

**What’s the agenda?:** So glad you asked! There isn’t one yet. Discussions leading up to the workshop will help set the expectations and requirements for what the IPFS projects need to accomplish in Q3. What will likely happen: Lectures, hacking, round table discussions about blockers, if you’ve been hankering to address a topic at large with this crew, now is the time!

**What we need from you:** Let us know what you’d like to see addressed by adding it to [this repository](https://github.com/ipfs/2016-Q3-Workshop), using [this issue template](https://github.com/ipfs/2016-Q3-Workshop/issues/3). We're going to be using that repo as our staging ground to talk about the event. Remember: Due to the nature of the structure of the event, not everything is guaranteed a slot.

**Questions comments concerns complaints? Let us know!** :tada: :heart:

---

Written by @em-ly, @jbenet and me. :D

**Comment by condiosluzverde** [comment]
+1 attendance remote during non-business hours US ET (NYC) - want to learn and maybe hack on something small atop lpfs api.

**Comment by aistich** [comment]
Hello everyone!
 You can RSVP for coming to the event in person or remotely at the link below.

https://www.eventbrite.com/e/ipfs-workshop-q3-and-beyond-tickets-26559316631?utm-medium=discovery&utm-campaign=social&utm-content=attendeeshare&utm-source=cp&utm-term=listing

**Comment by whereswaldon** [comment]
What time zone are the times on the tickets in? They say 10:00am - 5:00pm, but is that in Lisbon time? I'm in US ET (NYC). Is that 3:00pm-10:00pm for me?

**Comment by Kubuxu** [comment]
You can use this: http://www.worldtimebuddy.com/west-to-edt-converter

**Comment by VictorBjelkholm** [comment]
I'll be there as well! Coming from Barcelona, land in Lisbon at Saturday 23th and I leave on Wednesday 27th!

Looking forward to meeting you all!

**Comment by SShrike** [comment]
Oh I'd love to go in person, but I'm 19.5 thousand KM away :(

Hope the online stuff all gets up and going.
